website/client/js/services/userServices.js

```diff
@@ -151,7 +151,7 @@ angular.module('habitrpg')
         $http({
           method: method,
           url: url + queryString,
-          body: body,
```

Comment:
Hmm actually I'm not sure about this.. I mean, it makes sense but it's used for a lof of ops and we should have noticed if they didn't work.. @TheHollidayInn